### PR TITLE
Spec: describe JSON fields

### PR DESCRIPTION
**Pull request by lbam**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -32,13 +32,12 @@ is to:
 
 * activate the environment: `source dvenv/bin/activate`
 * change to the `docs` directory: `cd docs`
-* and run the bikeshed command: `bikeshed spec ams-schema-spec.bs ams-schema-spec.html`
+* run `make`.
```
Comment (jjmurre) [inline docs/README.md:35]: Nice!

File: docs/Makefile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ams-schema-spec.html: ams-schema-spec.bs
+	bikeshed spec $< $@
+	sed -i 's/ *$$//' $@
```
Comment (lbam) [inline docs/Makefile:3]: This does almost the same as the `whitespace` task from pre-commit. It's meant to make `git add -p` less painful.

Comment (jjmurre) [reply docs/Makefile:3]: 👍🏼

**Review by jjmurre — APPROVED** [review]
LGTM

**Pull request merged**